acceptance/clients/clients.go

```diff
@@ -436,6 +437,22 @@ func NewWafV1Client() (*golangsdk.ServiceClient, error) {
 	return openstack.NewWAFV1(client, golangsdk.EndpointOpts{Region: cloud.RegionName})
 }
 
+// NewCsbsV1Client returns authenticated CSBS v1 client
+func NewCsbsV1Client() (*golangsdk.ServiceClient, error) {
```

Comment:
Please use `osEnv.AuthenticatedClient()` so test can be run both with `OS_` vars and `clouds.yaml`